src/borg/archiver.py

```diff
@@ -2733,6 +2733,8 @@ def build_parser(self):
             'a_status_oddity': '"I am seeing ‘A’ (added) status for a unchanged file!?"',
             'separate_compaction': '"Separate compaction"',
             'list_item_flags': '"Item flags"',
+            'key_files': 'https://borgbackup.readthedocs.io/en/master/internals/data-structures.html#key-files',
```

Comment:
why is this a full https url? the other stuff here looks rather like some link text on the rhs.

Reply:
IMHO links here would be more helpful. Example:
```sh
borg help compact | tail --lines=1
See "Separate compaction" in Additional Notes for more details.
```
This raises a question: where to find "Additional Notes"?

I suppose it could read "See in Usage -> Additional Notes -> Separate compaction in the online documentation for more details.", but a URL that just take you directly there is better.

I'll remove the URL for now for consistency.